### ISSUE TITLE
add possibility to adjust learning rate of optimizer

### ISSUE THREAD

**Issue by Bartzi** [HPI-DeepLearning/ssnn#26]
we should have something that allows us to linearly/exponentially shift the learning rate of the optimizer in the training loop
